src/sass/app_2/_common/components/contracts_popup.scss

```diff
@@ -4,7 +4,7 @@ div.contracts-popup-container {
     background: $COLOR_WHITE;
     border: solid 1px $COLOR_GRAY;
     border-radius: 4px;
-    transition: transform 0.3s opacity 0.3s;
+    transition: transform 0.3s, opacity 0.3s, box-shadow 0.18s;
```

Comment:
Personally I think we rather not use `transition` on `box-shadow` since it re-paints on every frame, having a performance impact on the page.